wiki/markdown/query-aggregated-objects.md

```diff
@@ -1,14 +1,16 @@
 # Query Aggregated Objects
 
-Query aggregated objects can be done using two endpoints.
+Query aggregated objects can be done using the following endpoint.
+     
+    http://<host address>:<host port>/aggregated-objects/
 
-## The first endpoint uses several criteria
+## The first  option uses several criteria with the endpoint
 
-    http://<host adress>:<host port>/query
+    http://<host address>:<host port>/aggregated-objects/query
 
 1. The criteria can be passed as URL parameter in a get request
 
-        http://<host adress>:<host port>/query?request="testCaseExecutions.testCase.verdict:PASSED,testCaseExecutions.testCase.id:TC5,id:6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43"
+        http://<host adress>:<host port>/aggregated-objects/query?request="testCaseExecutions.testCase.verdict:PASSED,testCaseExecutions.testCase.id:TC5,id:6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43"
```

Comment:
This is a get query and that is not possible to do? I would want to see this kind of query as an option to really use.